enforcer/src/main/gen/org/wso2/gateway/discovery/config/enforcer/AuthHeaderProto.java

```diff
@@ -0,0 +1,53 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: wso2/discovery/config/enforcer/auth_header.proto
+
+package org.wso2.gateway.discovery.config.enforcer;
```

Comment:
I wonder why these files are generated specifically for your case?
Since you have not done any changes to protobuf files.

Reply:
@ShalkiWenushika can you get a pull and rebuild? This should be fixed in https://github.com/wso2/product-microgateway/pull/2013

Reply:
@menakaj my local branch is up to date.

Reply:
Removed these generated files